obsoleted by NIC_replug.patch

reverted 95fb8bb3181bbe1ee87c95e91dff94f74f148c33
as per: https://bugzilla.kernel.org/show_bug.cgi?id=202103#c20
diff --git a/drivers/net/phy/phy.c b/drivers/net/phy/phy.c
index 80be4d691e5b..4f9cc05decfd 100644
--- a/drivers/net/phy/phy.c
+++ b/drivers/net/phy/phy.c
@@ -866,8 +866,8 @@ void phy_state_machine(struct work_struct *work)
 		if (phydev->link) {
 			phydev->link = 0;
 			phy_link_down(phydev, true);
+			do_suspend = true;
 		}
-		do_suspend = true;
 		break;
 	}
 
